### PR TITLE
windows: Two simple errors using Add-Warning

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
I noticed these before, finally nailed them...

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

win_regedit and win_lineinfile

##### ANSIBLE VERSION

v2.4

### CODE REVIEW

**Comment by dagwieers** [comment]
@nitzmahone @jborean93 @jhawkesworth Easy one to merge.

File: lib/ansible/modules/windows/win_lineinfile.ps1 [hunk 1]
```diff
@@ -421,7 +421,7 @@ If ($state -eq "present") {
 	}
 
 	If ($insertbefore -and $insertafter) {
-		Add-Warning('Both insertbefore and insertafter parameters found, ignoring "insertafter=$insertafter"');
+		Add-Warning $result 'Both insertbefore and insertafter parameters found, ignoring "insertafter=$insertafter"'
```
Comment (jborean93) [inline lib/ansible/modules/windows/win_lineinfile.ps1]: Are you able to change the message to be `"` double quotes. The variable `$insertafter` won't actually be shown in this example. Would also be good to use `-obj` and `-message` for the parameters.

File: lib/ansible/modules/windows/win_regedit.ps1 [hunk 2]
```diff
@@ -45,7 +45,7 @@ if ($diff_support) {
 
 # Fix HCCC:\ PSDrive for pre-2.3 compatibility
 if ($path -match "^HCCC:\\") {
-    Add-Warning("Please use path: HKCC:\... instead of path: $path")
+    Add-Warning $result "Please use path: HKCC:\... instead of path: $path"
```
Comment (jborean93) [inline lib/ansible/modules/windows/win_regedit.ps1:48]: If you are changing the above can you add the parameters?

**Review by jborean93 — CHANGES_REQUESTED** [review]
Just 2 minor things to change.

File: lib/ansible/modules/windows/win_lineinfile.ps1 (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/modules/windows/win_lineinfile.ps1]: I am torn for the options though. We don't do it for Exit-Json or Fail-Json, and this one is in the same category.

Comment (jborean93) [reply lib/ansible/modules/windows/win_lineinfile.ps1]: Happy to merge it now (apart from the quoting side), we can always revisit it as yes you are correct there seems to be different ways we are doing this stuff.

Comment (dagwieers) [reply lib/ansible/modules/windows/win_lineinfile.ps1]: Thanks, I fixed the quotes already.

**Review by jborean93 — APPROVED** [review]

**Comment by jborean93** [comment]
LGTM, merging the changes into devel

**Pull request merged**